README.md

```diff
@@ -96,3 +102,15 @@ Run:
 ```
 go test ./..
 ```
+
+### Build contracts
+Example of generating go bindings for Ethereum contracts
+```
+ifcli build_contracts -e ${ABIGEN_BINARY}
```

Comment:
Chainlink's main repo already has abigen in `tools/bin/abigen`, maybe we can reuse it after merging repos?